mongodb-compass-community.json

```diff
@@ -0,0 +1,32 @@
+{
+    "homepage": "https://www.mongodb.com/download-center?jmp=hero#compass",
+    "version": "1.15.1",
+    "architecture": {
+        "64bit": {
+            "url": [
+                "https://downloads.mongodb.com/compass/mongodb-compass-community-1.15.1-win32-x64.zip"

```

Comment:
This can be simplified to:
```json
    "architecture": {
        "64bit": {
            "url": "https://downloads.mongodb.com/compass/mongodb-compass-community-1.15.1-win32-x64.zip",
            "hash": "D2B4B9AEF4E1DB53B7D367398084CD8D10628195D3FF098866EDA4CBE323C906"
        }
    },
```